drivers/bridge/bridge_test.go

```diff
@@ -72,6 +72,71 @@ func TestCreateFail(t *testing.T) {
 	}
 }
 
+func TestCreateMultipleNetworks(t *testing.T) {
+	defer netutils.SetupTestNetNS(t)()
+	d := newDriver()
+	dd, _ := d.(*driver)
+
+	config1 := &networkConfiguration{BridgeName: "net_test_1", AllowNonDefaultBridge: true}
+	genericOption := make(map[string]interface{})
+	genericOption[netlabel.GenericData] = config1
+	if err := d.CreateNetwork("1", genericOption); err != nil {
+		t.Fatalf("Failed to create bridge: %v", err)
+	}
+
+	config2 := &networkConfiguration{BridgeName: "net_test_2", AllowNonDefaultBridge: true}
+	genericOption[netlabel.GenericData] = config2
+	if err := d.CreateNetwork("2", genericOption); err != nil {
+		t.Fatalf("Failed to create bridge: %v", err)
+	}
+
+	config3 := &networkConfiguration{BridgeName: "net_test_3", AllowNonDefaultBridge: true}
+	genericOption[netlabel.GenericData] = config3
+	if err := d.CreateNetwork("3", genericOption); err != nil {
+		t.Fatalf("Failed to create bridge: %v", err)
+	}
+
+	// Verify the network isolation rules are installed, each network subnet should appear 4 times
+	verifyV4INCEntries(dd.networks, 4, t)
+
+	config4 := &networkConfiguration{BridgeName: "net_test_4", AllowNonDefaultBridge: true}
+	genericOption[netlabel.GenericData] = config4
+	if err := d.CreateNetwork("4", genericOption); err != nil {
+		t.Fatalf("Failed to create bridge: %v", err)
+	}
+
+	// Now 6 times
+	verifyV4INCEntries(dd.networks, 6, t)
```

Comment:
This should have failed ideally (if the `iptables` config option is honored).
Prior 2 comments will handle this case.
I think it will be a good test to have iptables disabled and make sure we don't install these rules.